extensions/wikia/MultiTasks/SpecialMultiWikiFinder_body.php

```diff
@@ -122,7 +122,7 @@ function showResults( ) {
 				'pages',
 				array( 'page_latest, page_wikia_id, page_id, page_is_redirect' ),
 				array(
-					'page_title'		=> mb_strtolower($this->mPageTitle),
+					'page_title'		=> mb_strtoupper( mb_substr( $this->mPageTitle, 0, 1 ) ) . mb_substr( $this->mPageTitle, 1 ),
```

Comment:
Maybe let's use `Language::ucfirst` instead?

```
> echo $wgLang->ucfirst('foo')
Foo
```

Reply:
We don't know what language the input string is in.